stock_archive_constraint/models/product_product.py

```diff
@@ -10,43 +10,52 @@ class ProductProduct(models.Model):
 
     @api.constrains("active")
     def _check_active_stock_archive_constraint_stock_quant(self):
-        res = self.env['stock.quant'].search(
+        res = self.env["stock.quant"].search(
```

Comment:
This is pre-commit stuff, so it shouldn't be also in the migration to v13 commit. You should follow full migration guide.

Reply:
It's necessary to merge PR in 12.0 previously to use migration guideline.

Reply:
Not really, you just need to follow the steps about pre-commit stuff and commit apart. That's what I want.